KlingonAssistant/data/mem-03-D.xml

```diff
@@ -438,7 +438,7 @@ As a time element, {DaHjaj:n} precedes any adverbials.[1, sec. 6.7]</column>
       <column name="entry_name">Damu'</column>
       <column name="part_of_speech">n</column>
       <column name="definition">crease</column>
-      <column name="definition_de"></column>
+      <column name="definition_de">Falte (im Papier)</column>
```

Comment:
Does this need the "im Papier" qualifier in German? What can "Falte" mean which isn't the kind of crease/fold in paper?